app/res/layout/recording_fragment.xml

```diff
@@ -83,20 +83,35 @@
         android:layout_marginTop="15dp"
         android:layout_weight=".1">
 
+
+        <Button
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/dimagi/commcare-android/blob/3ce92dc2ce8196284a31c400a04f0f4d85f2f52e/app/res/layout/recording_fragment.xml#L87"></span>Buttons in button bars should be borderless; use <code>style="?android:attr/buttonBarButtonStyle"</code> (and <code>?android:attr/buttonBarStyle</code> on the parent)</td>
    </tr>
  </tbody>
</table>